### DISCUSSION TITLE
ISO-8601-compliant datetimes

### DISCUSSION THREAD

**Discussion by CF-FHB-X** [realiii/fudgeo#14] [General]
Hey there!

Just ran into an issue with the datetimes (TLDR: the datetimes that are getting written to the GeoPackage aren't quite ISO-8601 compliant, as they're missing the `T` between the date and the time). I see that there's [a fix](https://github.com/realiii/fudgeo/commit/7dd3553de1a65894620c8ae44a0413d408726c9d) that's been merged in, which is perfect! Just curious when there's going to be a new release on PyPI with that fix? I've got a workaround for our downstream code, but I'd much rather use the fixed package.

Thanks!
Richard

**Comment by jlhumber** [comment]
Hey Richard,

Thanks for the ping.  We've just pushed it up to pypi as version 0.2.1.

Jason

**Reply by CF-FHB-X** [reply]
Thanks as always for the lightning quick responses! Off to remove my workarounds! 👍
